test/system/src/test/java/io/pravega/test/system/BookieFailoverTest.java

```diff
@@ -189,22 +190,22 @@ public void bookieFailoverTest() throws ExecutionException, InterruptedException
             createReaders(clientFactory, readerGroupName, SCOPE, readerGroupManager, STREAM, NUM_READERS);
 
             // Give some time to create readers before forcing a bookie failover.
-            Exceptions.handleInterrupted(() -> Thread.sleep(WAIT_AFTER_FAILOVER_MILLIS));
+            Exceptions.handleInterrupted(() -> Thread.sleep(BOOKIE_FAILOVER_WAIT_MILLIS));
 
             // Scale down bookie.
             Futures.getAndHandleExceptions(bookkeeperService.scaleService(2), ExecutionException::new);
 
-            log.info("Sleeping for {} seconds.", WAIT_AFTER_FAILOVER_MILLIS / 1000);
-            Exceptions.handleInterrupted(() -> Thread.sleep(WAIT_AFTER_FAILOVER_MILLIS));
+            log.info("Sleeping for {} seconds.", BOOKIE_FAILOVER_WAIT_MILLIS / 1000);
+            Exceptions.handleInterrupted(() -> Thread.sleep(BOOKIE_FAILOVER_WAIT_MILLIS));
 
             long writeCountBeforeSleep  = testState.getEventWrittenCount();
-            log.info("Write count is {} after {} seconds sleep after bookie failover.", writeCountBeforeSleep, WAIT_AFTER_FAILOVER_MILLIS / 1000);
+            log.info("Write count is {} after {} seconds sleep after bookie failover.", writeCountBeforeSleep, BOOKIE_FAILOVER_WAIT_MILLIS / 1000);
 
-            log.info("Sleeping for {} seconds.", WAIT_AFTER_FAILOVER_MILLIS / 1000);
-            Exceptions.handleInterrupted(() -> Thread.sleep(WAIT_AFTER_FAILOVER_MILLIS));
+            log.info("Sleeping for {} seconds.", BOOKIE_FAILOVER_WAIT_MILLIS / 1000);
+            Exceptions.handleInterrupted(() -> Thread.sleep(BOOKIE_FAILOVER_WAIT_MILLIS));
 
             long writeCountAfterSleep  = testState.getEventWrittenCount();
-            log.info("Write count is {} after {} seconds sleep after bookie failover.", writeCountAfterSleep, WAIT_AFTER_FAILOVER_MILLIS / 1000);
+            log.info("Write count is {} after {} seconds sleep after bookie failover.", writeCountAfterSleep, BOOKIE_FAILOVER_WAIT_MILLIS / 1000);
```

Comment:
this log seems incorrect, I mean by the time we are checking the write count its already 2* (BOOKIE_FAILOVER_WAIT_MILLIS/1000). Can you just correct it, please?

Reply:
You are right, fixed.